cmd/crictl/sandbox.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runPodSandboxSpec string
+var podSandboxID string
+var filter string
+
+var runtimeRunPodSanboxCommand = cli.Command{
+	Name:  "runsandbox",
+	Usage: "run a sandbox",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "podsandbox-spec",
+			Usage:       "Path of podsandbox spec file",
+			Destination: &runPodSandboxSpec,
+		},
+	},
+	Action: func(context *cli.Context) error {
+		// Set up a connection to the server.
+		conn, err := getRuntimeClientConnection(context)
+		if err != nil {
+			return fmt.Errorf("failed to connect: %v", err)
+		}
+		defer conn.Close()
+		client := pb.NewRuntimeServiceClient(conn)
```

Comment:
I think we can put these duplicated code in a new file, such as `util.go` or something you like.